operations/observability/mixins/workspace/rules/components/nodes/alerts.libsonnet

```diff
@@ -22,6 +22,19 @@
             },
             expr: 'nodepool:node_load1:normalized{nodepool=~".*workspace.*"} > 3',
           },
+          {
+            alert: 'AutoscalerAddsNodesTooFast',
+            labels: {
+              severity: 'critical',
+            },
+            'for': '1m',
```

Comment:
Why is duration needed in this condition?

@ArthurSens wdyt?

Reply:
technically not needed.

Reply:
Cool, let's see if we can remove? You can check to see if it [passes validation](https://github.com/gitpod-io/gitpod/tree/main/operations/observability/mixins#Rules-and-Alerts-validation).

Reply:
and done!